index.bs

```diff
@@ -6668,7 +6674,7 @@ In effect, where <var ignore>x</var> is a <emu-val>Number</emu-val> value,
 
     1.  Let |x| be [=?=] [=ToNumber=](|V|).
     1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
-        then [=ECMAScript/throw=] a <emu-val>TypeError</emu-val>.
+        then [=ECMAScript/throw=] a {{TypeError}}.
```

Comment:
Speaking of that, I'm not too clear on the difference between [ES/throw](https://heycam.github.io/webidl/#ecmascript-throw) and [ES binding throw](https://heycam.github.io/webidl/#es-creating-throwing-exceptions) -- they seem to be equivalent IMO.